docs/source/distributed_runner.rst

```diff
@@ -0,0 +1,45 @@
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+Running simulations in parallel
+===============================
+
+Once you successfully create a simulation specification and branch file it is time to use the distributed runner.
+Recall, we ran a single simulation with the model specification file in this way
+
+.. code-block:: console
+
+    simulate run /path/to/your/model/specification
+
+Very similar to this, ``vivarium-cluster-tools`` includes a command for simulating in parallel
+
+.. code-block:: console
+
+    psimulate run /path/to/your/model/specification  /path/to/your/branch
+
+By default, output will be saved in ``/share/scratch/users/{$USER}/vivarium_results``. If you want to save the
+results somewhere else you can specify your output directory as an optional argument
+
+.. code-block:: console
+
+    psimulate run /path/to/your/model/specification /path/to/your/branch -o /path/to/output
+
```

Comment:
Oh one thought: maybe add a sentence saying the runner will create a folder named the time your run starts in that output directory? I had a few questions about whether if you give two distributed runs the same output path, will they overwrite outputs

Reply:
that's a good point